tests/build/build_templates.go

```diff
@@ -687,6 +689,10 @@ var _ = framework.BuildSuiteDescribe("Build templates E2E test", Label("build",
 					})
 				}
 			})
+
+			It("should push Dockerfile to registry", Label(buildTemplatesTestLabel), func() {
```

Comment:
will be good to do this check before.. just after [this](https://github.com/konflux-ci/e2e-tests/blob/0612222491e1dd0679c571247bff97c594b070da/tests/build/build_templates.go#L346) check?

Reply:
Updated.